react/features/base/dialog/components/native/styles.js

```diff
@@ -45,8 +49,7 @@ export const bottomSheetStyles = {
 
     sheetItemContainer: {
         flex: -1,
-        maxHeight: '60%',
-        paddingHorizontal: MD_ITEM_MARGIN_PADDING
+        maxHeight: '60%'
```

Comment:
Should we raise this to lioke 90% now maybe?

Reply:
Or completely remove the maxHeight?

Reply:
Yeah, let's do that, now that there is a nice way to collapse the thing.